### DISCUSSION TITLE
Default value during form view [Jun 27]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2555] [Discord Threads]
> 🙋‍♂️ **Shard**

**Comment by o1lab** [comment]
> 🙋‍♂️ **FrancyMak**  
#### Hi, unfortunately it is not possible at the moment.   
####    
#### Some column types, though, may have default values.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/990566857595584512)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
